index.js

```diff
@@ -141,3 +129,29 @@ function getQueryString (search, reqUrl, opts) {
 function headersNoOp (headers) {
   return headers
 }
+
+function getCacheStorage (size) {
+  if (size === 0) {
```

Comment:
Does this means that the default caching of the URL is off?

Caching URLs is one of the most important things related to performance this package do. Generating those URL is relatively costly, and we should cache them as much as possible. I would _increase_ the cache size, not disable it.
Without caching, we can't really call it "fast".

Reply:
Hi @mcollina, totally agreed on your reminder. 

Default cache size remains at 100, as described here: https://github.com/fastify/fast-proxy/pull/26/files#diff-168726dbe96b3ce427e7fedce31bb0bcR138
Only when the `cacheURLs` value is explicitly `0`, we completely disable the cache storage interaction. This could make sense on certain scenarios were url patterns are not repetitive.